### PR TITLE
Update the friendship class section

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR updates the friendship class in API references section.

### CODE REVIEW

**Comment by shraddhavp** [comment]
@proudofsimin  @Rohitesh-Kumar-Jain  @huan  @Soumi7

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/api/friendship.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-## Friendship
+[Examples or Friend-Bot](https://github.com/wechaty/wechaty/blob/1523c5e02be46ebe2cc172a744b2fbe53351540e/examples/friend-bot.ts)
 
-Send, receive friend request, and friend confirmation events.
+## Global class Friendship
 
-1. send request
-2. receive request\(in friend event\)
-3. confirmation friendship\(friend event\)
+The Friendship class allows you to do the following fuctionalities.
```
Comment (Soumi7) [inline docusaurus/docs/api/friendship.md:12]: Fuctionalities -> functionalities

**Review by Soumi7 — CHANGES_REQUESTED** [review]

File: docusaurus/docs/api/friendship.md (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/api/friendship.md:12]: done

**Review by Soumi7 — APPROVED** [review]
Lgtm

**Review by proudofsimin — APPROVED** [review]

**Pull request merged**